### PR TITLE
Fix cow sale toast when no cows are salable

### LINKED ISSUE #48: Toast says "Cow Sold" when no cow was sold
# Overview

When selling cows, if you run out of cows, the app doesn't let you continue to sell cows. However, the toast message that pops up says "Cow Sold" even if you didn't just sell a cow.

Perhaps the toast should NOT pop up when you successfully sell a cow (since you already see the number of cows go up, and your money go down), but should ONLY pop up when you are out of cows to sell and try to buy a cow.. and in that case, the message should indicate that.

# Steps to reproduce

1. Join a commons
2. Try to sell cows until you run out of cows. See toasts that say "Cow Sold"
3. Try to sell another cow

- **Expected**: a toast that says something like: "You have already sold all of your cows"
- **Observed**: a toast that says "Cow Sold"

### PR DESCRIPTION
**Pull request by ethwu**
- Change the toast message to reflect the fact that no cows are sold
  when a user's cow count is zero
- Add tests for cow sale when cow count is greater than zero and when
  cow count is zero

Before:
Sell Toast Bad

After:
Sell Toast Good

Closes #48

### CODE REVIEW

**Review by minhbui27 — APPROVED** [review]
lgtm

**Review by RAILGUN1124 — APPROVED** [review]
lgtm

**Review by blakerandle — APPROVED** [review]
lgtm

File: frontend/src/tests/pages/PlayPage.test.js [hunk 1]
```diff
@@ -15,31 +18,37 @@ jest.mock("react-router-dom", () => ({
...
+    const sampleCommons = commonsFixtures.oneCommons[0];
+
+    // Expected toast strings for cow purchase.
+    const COW_BUY_TOAST = "Cow bought!";
+    // Expected toast string for when a cow purchase fails.
+    const COW_BUY_FAILURE_TOAST = "You can't buy a cow because you don't have enough money";
+    // Expected toast strings for cow sale.
+    const COW_SELL_TOAST = "Cow sold!";
+    // Expected toast string for when a cow sale fails.
+    const COW_SELL_FAILURE_TOAST = "You have already sold all of your cows";
```
Comment (pconrad) [inline frontend/src/tests/pages/PlayPage.test.js:41]: This is really nice. Well done.

**Review by pconrad — APPROVED** [review]
LGTM.. will test on qa before merging
